public/sass/_variables.light.scss

```diff
@@ -265,15 +262,15 @@ $success-text-color: lighten($green, 10%);
 $alert-error-bg: linear-gradient(90deg, $lobster-base, $lobster-shade);
 $alert-success-bg: linear-gradient(90deg, $green-base, $green-shade);
 $alert-warning-bg: linear-gradient(90deg, $lobster-base, $lobster-shade);
-$alert-info-bg: $sapphire-base;
+$alert-info-bg: linear-gradient(100deg, $sapphire-base, $sapphire-shade);
 
 // popover
 $popover-bg: $page-bg;
 $popover-color: $text-color;
 $popover-border-color: $gray-5;
 $popover-shadow: 0 0 20px $white;
 
-$popover-help-bg: $sapphire-base;
+$popover-help-bg: $btn-secondary-bg;
```

Comment:
any reason this is using the btn variable ?

Reply:
Hmm, no. Not sure what I was thinking there.